### ISSUE TITLE
UBT, improve documentation L102, L133, L134, L168

### ISSUE THREAD

**Issue by sarvesh-ost** [OpenST/JLP#82]

**Comment by jasonklein** [comment]
@sarvesh-ost: are you working on this or are you able to provide more specifics? When I look at those lines in https://github.com/OpenST/brandedtoken-contracts/blob/develop/contracts/UtilityBrandedToken.sol, it is not clear to me what improvements should be made.

**Comment by sarvesh-ost** [comment]
No, I am not working on this. I created this ticket to work on this. But like you, I couldn't understand this. Maybe @benjaminbollen / @gulshanvasnani  can clarify 🤔

**Comment by sarvesh-ost** [comment]
> @sarvesh-ost: are you working on this or are you able to provide more specifics? When I look at those lines in https://github.com/OpenST/brandedtoken-contracts/blob/develop/contracts/UtilityBrandedToken.sol, it is not clear to me what improvements should be made.

No, I am not working on this. I created this ticket to work on this. But like you, I couldn't understand this. Maybe @benjaminbollen / @gulshanvasnani can clarify 🤔

**Comment by jasonklein** [comment]
@benjaminbollen: it is not clear what improvements are to be made to the UBT documentation, per JLP Issue 80, "UBT, improve documentation L102, L133, L134, L168".

Looking at [UBT before the post-deep-dive changes](https://github.com/OpenST/brandedtoken-contracts/blob/0669ae670bbeed14af3e9d79619a129dfcaba3c4/contracts/UtilityBrandedToken.sol), the items I suspect were to be improved appear to have been either improved or obviated by @pgev's changes. 

Please confirm whether this ticket remains to be resolved or if it can be closed.
